build-it/stage_artifacts.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import logging
+import os
+from pathlib import Path
+import sys
+import tarfile
+from zipfile import ZipFile
+import shutil
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.DEBUG)
+
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+_logger.addHandler(handler)
+
+def main():
+    args = parse_args()
+    _logger.debug(f"Artifacts path is {os.listdir(args.downloaded_path)}")
+    artifact_folders = [os.path.join(args.downloaded_path, x) for x in os.listdir(args.downloaded_path)]

```

Comment:
add a comment
// args.downloaded_path contains artifact folders(each artifact is downloaded as a folder of its own)